client/my-sites/plans/main.jsx

```diff
@@ -143,11 +144,14 @@ class Plans extends Component {
 			);
 		}
 
+		const hideFreePlan =
+			! isEnabled( 'onboarding/2023-pricing-grid' ) || ! isFreePlan( currentPlan.productSlug );
```

Comment:
```suggestion
const hideFreePlan = isEnabled('onboarding/2023-pricing-grid') 
    ? ! isFreePlan(currentPlan.productSlug) 
    : true;
```
I find the logical OR with negations on both sides a little hard to follow. The ternary form feels a little more concise and easier to read. The two should be identical?

Reply:
This version works too. Thanks!